packages/dev/docs/pages/releases/2022-06-15.mdx

```diff
@@ -0,0 +1,239 @@
+
+In addition to these components is a new library [`@internationalized`](https://react-spectrum.adobe.com/internationalized/index.html), that provides locale-aware objects and functions for manipulating dates and times. This library supports time zone and calendar system aware arithmetic, locale-specific queries and conversions, and more in just 8 kB! Learn more about it in our [docs](https://react-spectrum.adobe.com/internationalized/date/index.html).
+
+A special thanks to [@devongovett](https://github.com/devongovett) for his work on all things date and to our other contributors for this release 🤩
+
+Enjoy!
+
+
+## New Components
+React Spectrum
+  - [Calendar](https://react-spectrum.adobe.com/react-spectrum/Calendar.html)
+  - [RangeCalendar](https://react-spectrum.adobe.com/react-spectrum/RangeCalendar.html)
+  - [DateField](https://react-spectrum.adobe.com/react-spectrum/DateField.html)
+  - [DatePicker](https://react-spectrum.adobe.com/react-spectrum/DatePicker.html)
+  - [DateRangePicker](https://react-spectrum.adobe.com/react-spectrum/DateRangePicker.html)
+  - [TimeField](https://react-spectrum.adobe.com/react-spectrum/TimeField.html)
+
+React Aria
+  - [useCalendar](https://react-spectrum.adobe.com/react-aria/useCalendar.html)
+  - [useRangeCalendar](https://react-spectrum.adobe.com/react-aria/useRangeCalendar.html)
+  - [useDateField](https://react-spectrum.adobe.com/react-aria/useDateField.html)
+  - [useDatePicker](https://react-spectrum.adobe.com/react-aria/useDatePicker.html)
+  - [useDateRangePicker](https://react-spectrum.adobe.com/react-aria/useDateRangePicker.html)
+  - [useTimeField](https://react-spectrum.adobe.com/react-aria/useTimeField.html)
+
+
+## Fixed
+* Update TS for strict mode in `SpectrumButton` - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3143)
+* Allow null in return types of `KeyboardDelegate` in collections - [@dlech](https://github.com/dlech) - [PR](https://github.com/adobe/react-spectrum/pull/3139)
+
+## Docs
+* Update W3C APG links - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/3204)
+* Update TableView docs for checkbox/highlight + onAction behavior update - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/3205)
+
+## Under construction
+
+Fixes to pre-released components are listed below. Please feel free to try them out, and report any issues you encounter.
+
+* Fix `ListView` shifting height with truncation - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/3158)
+* Allow focusing disbaled rows when dragging in `ListView` - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/3181)
```

Comment:
sorry bout the PR title, this fix actually prevents focus from reaching the disabled rows if they aren't valid drop targets. Perhaps this should be "ListView drag and drop focus fixes" or something along those lines?